cpp/tests/filling/sequence_tests.cpp

```diff
@@ -134,3 +137,52 @@ TYPED_TEST(SequenceTypedTestFixture, DefaultStep)
 
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(*result, expected_w);
 }
+
+TEST_F(SequenceTestFixture, DateSequenceBasic)
+{
+  // Timestamp generated using https://www.epochconverter.com/
+  timestamp_scalar<timestamp_s> init(1629852896L, true);  // 2021-08-25 00:54:56 GMT
+  size_type size{5};
+  size_type months{1};
+
+  fixed_width_column_wrapper<timestamp_s, int64_t> expected{
+    1629852896L,  // 2021-08-25 00:54:56 GMT
```

Comment:
Is there a test utility to create time stamps? Perhaps something that can exactly parse the string in your comment?
```suggestion
    make_timestamp("2021-08-25 00:54:56 GMT"),
```
The function could simply assert if the format did not match exactly.